cmd/tailscaled/tailscaled_windows.go

```diff
@@ -264,6 +271,15 @@ func startIPNServer(ctx context.Context, logid string) error {
 	return err
 }
 
+func handleSessionChange(chgRequest svc.ChangeRequest) {
+	if chgRequest.Cmd != svc.SessionChange || chgRequest.EventType != windows.WTS_SESSION_UNLOCK ||
+		!flushDNSOnSessionUnlock {
+		return
+	}
+
```

Comment:
should we log here? or does dns.Flush already log? Even if it does, it doesn't log _why_ it's being flushed, so maybe we should log here anyway.

Reply:
I added a log entry.